app/controllers/api/events_controller.rb

```diff
@@ -6,5 +6,15 @@ def past
       events = Event.where('date < ?', DateTime.current).order(date: :desc)
       render status: 200, json:  { data: events.as_json }
     end
+
+    def past_by_month
+      event_date = DateTime.new(params[:year].to_i, params[:month].to_i)
+      events = Event.where(date: event_date..event_date.end_of_month)
```

Comment:
My one comment is that I think we should make an assumption that there will only be one event every month (which we can revisit if this changes, but I don't think it will anytime soon) -- could we use `.first` to return only one event?

Reply:
Thanks heaps @egiurleo, I'll get onto that this weekend.